PIPELINE.md

```diff
@@ -0,0 +1,109 @@
+# Jenkins Pipeline 
+
+## Overview
+
+The Jenkinsfile in this repository contains a sample pipeline that performs a 
+build and applies object configurations to instantiate this application.
+
+The pipeline can be used to maintain a running application on an OpenShift
+cluster.
+
+## Basic: Getting Started
+
+1. Start an OpenShift cluster (origin v1.3 and later). Create a new project.
+2. [Optional] Create a fork of this repository
+3. Instantiate the pipeline from template by invoking:
+   ```
+   oc new-app -f https://raw.githubusercontent.com/FORK/parksmap-web/add-jenkinsfile/ose3/pipeline-buildconfig-template.json -p GIT_URI=https://github.com/FORK/parksmap-web.git
+   ```
+   
+   (Where FORK is your own fork of this repository or `openshift-roadshow`)
+4. Start the pipeline build from the CLI with:
+   ```
+   oc start-build parksmap-web-pipeline
+   ```
+
+   OR
+
+   Start the pipeline from the web console. Navigate to the project, Builds->Pipelines. 
+   Click `Start` next to the pipeline.
+
+
+## Advanced: Example of a PR-based development flow
+
+The Jenkinsfile in this repository can be used to support a PR flow with Jenkins on OpenShift.
+The main application is built and deployed to an 'integration' project. Every time a PR is
+submitted to the source repository, a new OpenShift project is created to build and test the
+merged PR and allow for manual testing. Once the manual testing is approved, the build succeeds.
```

Comment:
I think it would be good to provide more a summary of what the next steps are having you do, especially what the org plugin is going to do (scan your org and add jenkins jobs), the steps themselves don't provide much detail on why you're doing certain things or what the expected result is, so an overview would be helpful.